docs/shared/packages/gradle/gradle-plugin.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Overview of the Nx Gradle Plugin
+description: The Nx Gradle plugin contains generators and plugins that are useful for Gradle projects in an Nx workspace.
+---
+
+The Nx Gradle plugin contains generators and plugins that are useful for Gradle projects in an Nx workspace.
+
+## Setting Up @nx/gradle
```

Comment:
We should have a section here about setting up a new workspace with gradle + Nx. Like the Nest one: https://nx-dev-git-fork-xiongemi-feature-nxc-544-add-gradle-ce7275-nrwl.vercel.app/nx-api/next/documents/overview

We should have 2 versions:
* Gradle Init
* Spring Initializr